apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/proposals/[proposalId]/page.tsx

```diff
@@ -215,71 +219,61 @@ export default async function Proposal({
-
-          <h4 className="font-press">Pool: {poolId}</h4>
+    <div className="mx-auto flex max-w-6xl flex-col gap-10 p-1 sm:p-8">
+      <header className="section-layout flex flex-col items-start gap-10 sm:flex-row">
+        <div className="flex w-full items-center justify-center sm:w-auto">
+          <Image
+            src={proposalImg}
+            alt={`proposal image ${proposalIdNumber}`}
+            height={163}
+            width={164}
+          />
         </div>
-
-        {/* title - description - status */}
-        <div className="border2 relative space-y-12 rounded-xl bg-white px-8 py-4">
-          <div className="flex justify-end">
-            <Badge status={status} />
-          </div>
-          <div className=" flex items-baseline justify-end space-x-4 ">
-            <h3 className="w-full text-center text-2xl font-semibold">
-              {title}
-            </h3>
-          </div>
-          <div className="">
-            <p className="text-md text-justify">{description}</p>
-          </div>
+        <div className="flex w-full flex-col gap-8">
           <div>
-            {/* reqAmount - bene - creatBy */}
-            <div className="flex justify-between ">
-              {!isSignalingType && !!requestedAmount && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Requested Amount
-                  </span>
-                  <span className="flex items-center gap-2 text-lg">
-                    {formatTokenAmount(requestedAmount, 18)}{" "}
-                    <span>{tokenSymbol}</span>
-                  </span>
-                </div>
-              )}
-              {!isSignalingType && beneficiary && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Beneficiary
-                  </span>
-                  <EthAddress address={beneficiary} actions="copy" />
-                </div>
-              )}
-              {submitter && (
-                <div className="flex flex-1 flex-col items-center space-y-4">
-                  <span className="text-md font-bold underline">
-                    Created By
-                  </span>
-                  <EthAddress address={submitter} actions="copy" />
-                </div>
-              )}
+            <div className="mb-4 flex flex-col items-start gap-4 sm:mb-2 sm:flex-row sm:items-center sm:justify-between sm:gap-2">
+              <h2>
+                Proposal #{proposalIdNumber} - {title}
+              </h2>
+              <Badge type={proposalType} />
+            </div>
+            <div className="flex items-center justify-between gap-4 sm:justify-start">
+              <Badge status={status} />
+              <p className="font-semibold">
+                {prettyTimestamp(proposalData?.createdAt || 0)}
```

Comment:
```suggestion
                {prettyTimestamp(proposalData?.createdAt ?? 0)}
```
Nothing will change but its would just save an extra operation in the case createdAt is 0

Reply:
?? : Null ord Undefined will coalesce (use right part)
|| : All falsy values including null or undefined but also 0 and ""

Reply:
Need reason to resolve this :p